deploy/metrics-apiservice.yaml

```diff
@@ -0,0 +1,12 @@
+apiVersion: apiregistration.k8s.io/v1beta1
+kind: APIService
+metadata:
+  name: v1alpha1.metrics
+spec:
+  insecureSkipTLSVerify: true
+  group: metrics
```

Comment:
Let's change this as a part of moving api to beta (outside of the scope of this PR)